src/main/java/com/minecolonies/coremod/inventory/InventoryCitizen.java

```diff
@@ -1001,8 +1002,8 @@ public void dropAllItems()
 
                 if (!itemstack.isEmpty())
```

Comment:
Here is still an isEmpty() check.
Please replace that with the ItemStackUtil one.